tests/components/eafm/conftest.py

```diff
@@ -0,0 +1,30 @@
+"""eafm fixtures."""
+# pylint: disable=redefined-outer-name
+
+import datetime
+
+from asynctest import patch
+import pytest
+
+
+@pytest.fixture()
+def mock_get_stations():
```

Comment:
```suggestion
@pytest.fixture(name="mock_get_stations")
def mock_get_stations_fixture():
```

Reply:
This didn't seem to be needed to fix the lint, so i've left as is for now.